File: katas/8 kyu/reversed-sequence/solution.py
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def reverse_seq(n):
    return sorted(list(range(1 , n+1)),reverse=True)
# ------------------------------ CLEVER SOLUTION ----------------------------- #
def reverseseq(n):
    return list(range(n, 0, -1))
# ----------------------------------- TEST ----------------------------------- #
@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(reverse_seq(5),[5,4,3,2,1])

File: katas/8 kyu/reversed-sequence/solution.js
// reversed-sequence
/* ------------------------------- MY SOLUTION ------------------------------ */
const reverseSeq = n => {
  return [...Array(n).keys()].map(i => i+1).reverse();
};
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
const reverseSeq = n => {
  return Array(n).fill(0).map((e, i) => n - i );
};

const reverseSeq = n => {
  let arr = [];
    for (let i=n; i>0; i--) {
    arr.push(i);
    } return arr;
};
/* ---------------------------------- TEST ---------------------------------- */
// const chai = require("chai");
// const Test = chai.assert.strictEqual; // replace Test.assertEqual with Test
// chai.config.truncateThreshold=0;
const chai = require("chai");
const assert = chai.assert;
chai.config.truncateThreshold=0;

describe("reverseSeq", function() {
  it("Sample Test", function() {
    assert.deepEqual(reverseSeq(5), [5, 4, 3, 2, 1]);
  });
});